### PR TITLE
[MVI-1025] (requests-mv-integrations) build curl: POST include session auth

### PR DESCRIPTION
**Pull request by tulitune**
build curl: POST include session auth:
Use mv_request.session.cookies as request_auth, if not provided.

### CODE REVIEW

File: requests_mv_integrations/request_mv_integration.py [hunk 1]
```diff
@@ -909,6 +909,11 @@ def _request(
 
         try:
             if build_request_curl:
+                # In case no authentication information has been provided,
+                # use session's cookies information, if exists
+                if not request_auth and self.session.cookies and len(self.session.cookies) > 0:
+                    request_auth = self.session.cookies
+
```
Comment (jefftune) [inline requests_mv_integrations/request_mv_integration.py]: @tulitune 
Could `self.session` ever contain `requests.auth.HTTPBasicAuth`?

Comment (tulitune) [inline requests_mv_integrations/request_mv_integration.py]: @jefftune 
From examining the code, it appears that not.
The only usage of the session property in the MvRequest class, as a container, is for storing cookies information.
( We can see this, if we search for the string "mv_request.session" in the mv-integrations project ).

File: requests_mv_integrations/support/curl.py [hunk 2]
```diff
@@ -92,17 +92,26 @@ def command_line_request_curl(
...
-
-        header_basic_auth = {'Authorization': _basic_auth_str(username, password)}
-
-        if request_headers:
-            if 'Authorization' not in request_headers:
-                request_headers.update(header_basic_auth)
-        else:
-            request_headers = header_basic_auth
+    # Handle authentication info
+    request_cookies_str = None
```
Comment (jefftune) [inline requests_mv_integrations/support/curl.py]: Recommend renaming `request_cookies_str` to `request_cookie`.

File: requests_mv_integrations/support/curl.py [hunk 3]
```diff
@@ -114,6 +123,9 @@ def command_line_request_curl(
 
     command = "curl --verbose -X {request_method} -H {headers} --connect-timeout {timeout}"
 
+    if request_cookies_str:
```
Comment (jefftune) [inline requests_mv_integrations/support/curl.py]: Recommend renaming `request_cookies_str` to `request_cookie`.

**Pull request merged**